lib/stack_master/sparkle_formation/user_data_file.rb

```diff
@@ -0,0 +1,32 @@
+require 'sparkle_formation'
+require 'erubis'
+
+class SparkleFormation
+  module SparkleAttribute
+    module Aws
+      class SfEruby < Erubis::Eruby
+        include Erubis::ArrayEnhancer
+
+        def add_expr(src, code, indicator)
+          case indicator
+            when '='
+              src << " #{@bufvar} << (" << code << ');'
```

Comment:
Ok, I think I have a rough idea what this line does. But I wonder if this can be simplified a bit like:
```
src << " #{@bufvar} << #{code};"
```